beacon-chain/rpc/prysm/v1alpha1/validator/proposer_execution_payload.go

```diff
@@ -94,10 +95,21 @@ func (vs *Server) getExecutionPayload(ctx context.Context, slot types.Slot) (*en
 		SafeBlockHash:      parentHash,
 		FinalizedBlockHash: finalizedBlockHash,
 	}
+
+	feeRecipient := params.BeaconConfig().DefaultFeeRecipient
+	recipient, err := vs.BeaconDB.FeeRecipientByValidatorID(ctx, vIdx)
+	switch err == nil {
+	case true:
+		feeRecipient = recipient
+	case errors.As(err, kv.ErrNotFoundFeeRecipient): // If fee recipient is not found, use the default fee recipient.
+		logrus.WithError(err).WithField("validatorIndex", vIdx).Error("Fee recipient not found. Using default fee recipient.")
```

Comment:
wonder if you should print what the default fee recipient address is.

Reply:
done!